introduction.md

```diff
@@ -0,0 +1,2 @@
+Hi, I'm Nazaret and this is a practice of mastering in github of Codeschool.com
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.